manuscript/portal_iterative_forecasting_ms.Rmd

```diff
@@ -106,12 +106,14 @@ Each month, we present our forecasts on a website that displays monthly rodent f
 
+Expanding the community of ecological forecasters using continous analysis approaches will require both an expansion of the current toolkit and the development of standards to facilitate interoperability of forecasts and forecasting systems. One of the major challenges for our current forecasting system is supporting computationally intensive forecasts. Projects involving larger datasets and/or complex modelling approaches will require either hosted solutions that provide infrastructure for running continuous integration and allow long-running distributed jobs or solutions that involve the user setting up their own continuous analysis sytem on cloud infrastructure or high performance computing centers. Event-driven serverless cloud platforms like OpenWhisk (https://openwhisk.apache.org/) and AWS Lambda (https://aws.amazon.com/lambda/) provide potential as hosted solutions, and open source continuous integrations systems like Jenkins (https://jenkins.io/) can be integrated with either cloud or high performance computing centers. However both solutions are currently more complicated to set up than the hosted continuous integration approach we have employed using Travis. In addition to scalability issue for more computationally intensive projects, the toolkit for continuous analysis needs be made more researcher friendly. To broaden the user-base that can use continuous analysis for forecasting, we recommend the development of tools that make setting up continuous analysis easier by automating configuration steps. We also recommend the development of tools or data repository infrastructure to support the easy automated archiving of regularly generated data and forecasts [see @yenni2018]. Finally, the development of standards for ecological forecasting to allow interoperability among forecasting systems will be essential for the growth of this field [see discussions of data standards, meta-data, and ontologies in ecology more broadly @jones2006; @madin2008; @michener2012]. Now is an opportune time for developing these standards while the community of ecological forecasters is still small. While we have developed an initial format for storing and sharing forecasts it is still lacking in several areas. Most notably our approach to storing information on models and their associated uncertainty is insufficient for all but the simplest models. Improving this framework will require capturing both covariances between state variables and the full uncertainty in the models, by either storing full model objects or additional information like full ensembles of predictions (e.g., from Monte Carlo based approaches). This is challenging due to a lack of general standards for reporting uncertainty [@dietze2018].  
```

Comment:
Need a comma: "However, both solutions..."
And another: "...initial format for storing and sharing forecasts, it is still lacking..."
And I'd prefer another: "Most notably, our approach..."